lib/workers/repository/config-migration/branch/index.ts

```diff
@@ -18,10 +22,47 @@ export async function checkConfigMigrationBranch(
     return null;
   }
   const configMigrationBranch = getMigrationBranchName(config);
-  if (await migrationPrExists(configMigrationBranch)) {
+  const commitMessageFactory = new ConfigMigrationCommitMessageFactory(
+    config,
+    migratedConfigData.filename
+  );
+  const prTitle = commitMessageFactory.create().toString();
+  const closedPrConfig: FindPRConfig = {
+    branchName: configMigrationBranch,
+    prTitle,
+    state: PrState.Closed,
+  };
+  const branchConfig: BranchConfig = {
+    prTitle,
+    manager: '',
+    upgrades: [],
+    branchName: configMigrationBranch,
+    userStrings: {
+      ignoreTopic:
+        (config.ignoreTopic as string) ?? 'Renovate Ignore Notification',
```

Comment:
should we just hard code it instead? ignoreTopic is user configurable

Reply:
why do we need the string cast?

Reply:
TS2322: Type 'unknown' is not assignable to type 'string'.
TS2571: Object is of type 'unknown'.

ill just hardcode it, i dont see the benefit of making this configurable.